app/migrations/20240809141227-create-datasourceI18n.cjs

```diff
@@ -0,0 +1,50 @@
+"use strict";
+
+const sql_up = `create table if not exists public."DataSourceI18n"
+(
+    datasource_id uuid not null constraint "DataSourceI18n_pkey" primary key,
+    datasource_name            varchar(255),
```

Comment:
Is this column not translated?

Reply:
it isn't 
![image](https://github.com/user-attachments/assets/dc68ae88-0b16-43cd-889a-48873520a6bb)